commands/command_edit.cpp

```diff
@@ -1285,6 +1286,70 @@ void EditCylinder::undo()
 	redo();
 }
 
+EditSensors::EditSensors(int cylIndex, const std::vector<int> &changedSensors)
+	: dc(current_dc), d(current_dive)
+{
+	if (!d || !dc)
+		return;
+
+	setText(Command::Base::tr("Edit sensors"));
+
+	std::vector<int> allSensors(MAX_SENSORS);
```

Comment:
Just an extremely minor nit-pick: since `MAX_SENSORS` is a compile-time constant, one might prefer `std::array<int, MAX_SENSORS>` here.

Reply:
Ok disregard previous comment - you then erase from the container. vector is the correct container here.